recipes/mydbfinder/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "MyDbFinder" %}
+{% set version = "1.0.5" %}
+{% set sha256 = "284cb919cd92369776348c76210ff72be244ac35de07558d8638e2a34f4e65c0" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://bitbucket.org/genomicepidemiology/{{ name|lower }}/get/{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  host:
+    - git
```

Comment:
It looks like the build is just copying over files so I don't think `git` or `kma` are needed as host dependencies (these are the dependencies used just when building)

Reply:
Sorry I forgot about them when changing the build. They have now been removed.Thank you.